src/equations/compressible_euler_quasi_1d.jl

```diff
@@ -163,6 +163,10 @@ end
 """
 @inline function flux_nonconservative_chan_etal(u_ll, u_rr, orientation::Integer,
                                                 equations::CompressibleEulerEquationsQuasi1D)
+@inline function flux_nonconservative_chan_etal(u_ll, u_rr, normal_direction, 
+                                                equations::CompressibleEulerEquationsQuasi1D)
+@inline function flux_nonconservative_chan_etal(u_ll, u_rr, normal_ll, normal_rr,
+                                                equations::CompressibleEulerEquationsQuasi1D)
```

Comment:
```suggestion
    flux_nonconservative_chan_etal(u_ll, u_rr, orientation::Integer,
                                   equations::CompressibleEulerEquationsQuasi1D)
    flux_nonconservative_chan_etal(u_ll, u_rr, normal_direction, 
                                   equations::CompressibleEulerEquationsQuasi1D)
    flux_nonconservative_chan_etal(u_ll, u_rr, normal_ll, normal_rr,
                                   equations::CompressibleEulerEquationsQuasi1D)
```

Could you please update the other docstrings in this file, too? We shouldn't write in our docs that we use `@inline` here and it's clear that they are functions from the docs since Documenter.jl adds the "method" tag to them..

Reply:
Yes, thanks for catching that!